numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
I favor the latter, because later on you end up with stuff like this

``` python
b = np.array((a[25:0:-1] * -1).tolist() +
             list(a) +
             (99 + 100 - a[-2:-22:-1]).tolist())
```
from `TestReflect.test_check_odd_method`.

I'm really not sure that's better than spelling out the array.